internal/model/client.go

```diff
@@ -42,25 +45,26 @@ func (c Client) GetURL() string {
 	return fmt.Sprintf("/supervision/#/clients/%d", c.Id)
 }
 
-func (c Client) GetActiveOrders() []Order {
+func (c Client) GetActiveOrders(orderType string) []Order {
 	var activeOrders []Order
 	for _, order := range c.Orders {
-		if order.Status.Handle == "ACTIVE" {
+		if order.Status.Handle == "ACTIVE" && (orderType == "" || order.Type == orderType) {
 			activeOrders = append(activeOrders, order)
 		}
 	}
 	return activeOrders
 }
 
-func (c Client) GetMostRecentOrder(orders []Order) Order {
+func (c Client) GetMostRecentOrder(orders []Order, orderType string) Order {
 	var mostRecentlyMadeActiveOrder Order
 	var mostRecentlyMadeOrder Order
 
 	for _, order := range orders {
-		if order.MadeActiveDate.After(mostRecentlyMadeActiveOrder.MadeActiveDate) {
+		ofType := orderType == "" || order.Type == orderType
```

Comment:
I'm struggling with the name `ofType`, is there any alternative that would be clearer? We're checking the OrderType is not null right? Maybe something like `hasOrderType`?

Reply:
I wasn't sure about that name either haha 😆 
it's checking if the order is of the requested type `orderType` or if no specific type was requested.
`hasOrderType` would be misleading, as they all have order type. maybe `isRequestedOrderType`?